manifests/client.pp

```diff
@@ -16,8 +16,8 @@
   file { $localcacert:
     ensure  => present,
     content => $cert,
-    owner   => 0,
-    group   => 0,
+    owner   => $localcacert_owner,
```

Comment:
Do you think we need to do this or just omit the `owner` and `group` entirely?  Omitting seemed to work in my case as the application of the puppet settings catalog by the agent self-corrected the files to whatever owner it thinks the files need to have and I didn't end up with changed resources on every run when the files weren't actually changing.